assets/evaluation_on_cloud/environments/evaluations-built-in/context/online_eval/evaluate.py

```diff
@@ -101,6 +102,24 @@ def download_evaluators_and_update_local_path(evaluators):
     return evaluators
 
 
+def copy_evaluator_files(command_line_args):
```

Comment:
ideally this would be a shared module. Please create a follow up task under the larger tech debt task to remove copied code where it can be shared as a module